app/assets/javascripts/application.js.erb

```diff
@@ -457,15 +472,17 @@ $(document).ready(function() {
     }
 
     var replies = comment.nextAll(".comments").first();
-    $.get("/comments/" + comment.attr("data-shortid") + "/reply",
-    function(data) {
-      var reply = $($.parseHTML(data));
-      reply.attr("id", "reply_form_" + comment.attr("id"));
-      replies.prepend(reply);
-      var ta = reply.find("textarea");
-      ta.focus().text(sel);
-      autosize(ta);
-    });
+    var done = Lobsters.loading($(this), () => {
+      $.get("/comments/" + comment.attr("data-shortid") + "/reply", function(data) {
+        done()
```

Comment:
I'm confused by names here - is this calling the `done` defined on line 212 or 475? Maybe let's give them both more descriptive names like `commentFormSubmitted` to match their behavior.